installer/Migration/V5_4_0/permission/screens.yaml

```diff
@@ -0,0 +1,34 @@
+'View claim module':
```

Comment:
is this keys same as 4.x
we may have to insert these conditionally, because some 4.x instances may already have these rules.